src/main/java/org/opentripplanner/graph_builder/module/NetexModule.java

```diff
@@ -0,0 +1,90 @@
+package org.opentripplanner.graph_builder.module;
+
+import org.opentripplanner.calendar.impl.MultiCalendarServiceImpl;
+import org.opentripplanner.graph_builder.services.GraphBuilderModule;
+import org.opentripplanner.model.calendar.CalendarServiceData;
+import org.opentripplanner.model.impl.OtpTransitServiceBuilder;
+import org.opentripplanner.netex.loader.NetexBundle;
+import org.opentripplanner.netex.loader.NetexLoader;
+import org.opentripplanner.routing.edgetype.factory.GtfsStopContext;
+import org.opentripplanner.routing.edgetype.factory.PatternHopFactory;
+import org.opentripplanner.routing.graph.Graph;
+import org.opentripplanner.routing.impl.DefaultFareServiceFactory;
+import org.opentripplanner.routing.services.FareServiceFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.HashMap;
+import java.util.List;
+
+// TODO OTP2 - logging?
+
+/**
+ * This module is used for importing the NeTEx CEN Technical Standard for exchanging Public Transport schedules and
+ * related data (<a href="http://netex-cen.eu/">http://netex-cen.eu/</a>). Currently it only supports the Norwegian
+ * profile (<a href="https://enturas.atlassian.net/wiki/spaces/PUBLIC/">
+ * https://enturas.atlassian.net/wiki/spaces/PUBLIC/</a>), but it is intended to be updated later to support other
+ * profiles.
+ */
+public class NetexModule implements GraphBuilderModule {
+
+    private static final Logger LOG = LoggerFactory.getLogger(NetexModule.class);
+
+    private List<NetexBundle> netexBundles;
+
+    private FareServiceFactory fareServiceFactory = new DefaultFareServiceFactory();
+
+    public NetexModule(List<NetexBundle> netexBundles) {
+        this.netexBundles = netexBundles;
+    }
+
+    @Override
+    public void buildGraph(Graph graph, HashMap<Class<?>, Object> extra) {
+
+        graph.clearTimeZone();
+        MultiCalendarServiceImpl calendarService = new MultiCalendarServiceImpl();
+        GtfsStopContext stopContext = new GtfsStopContext();
+
+        try {
+            for (NetexBundle netexBundle : netexBundles) {
+                OtpTransitServiceBuilder daoBuilder = new NetexLoader(netexBundle).loadBundle();
+
+                calendarService.addData(daoBuilder);
```

Comment:
I agree, the steps are not clear, and the naming is confusing - this also apply to the `GtfsModule`. There is several problems. The `CalendarService#addData` is misleading, since this is *building/merging* calendar data. Passing in the builder is breaking the rule of only passing in *used dependencies*, here only the used *calendar and trip data* should be passed inn, not the everything.

I will extract the merging calendar data functionality out of `CalendarServiceImpl` into `CalendarServiceDataBuilder` and let OtpTransitServiceBilder delegate the merging of calendar data. Then the `daoBuilder` should have been renamed to ´transitBuilder´.